openlibrary/plugins/openlibrary/code.py

```diff
@@ -217,6 +217,26 @@ def default(self, obj):
         )
 
 
+class team(delegate.page):
+    path = '/about/team'
+
+    def GET(self):
+        with Path('/openlibrary/openlibrary/templates/about/team.json').open(
+            mode='r'
+        ) as f:
+            team_members: list[dict[str, str]] = sorted(
+                json.load(f), key=lambda member: member['name'].split()[-1]
+            )
+            return render_template("about/index.html", team_members=team_members)
```

Comment:
@cclauss, will the access to this endpoint be automatically cached somewhere outside of Python such that this file isn't opened every time someone visits this page?

Reply:
Let's use the `openlibrary.core.cache.memoize` function decorator to remember (in memcache) the contents of the file for up to 5 minutes like
https://github.com/internetarchive/openlibrary/blob/afc9cc36d75873254f321cd66a7a55c56260c891/openlibrary/core/admin.py#L67-L69

Create a function:
```python
# Cache the team_members in memcache for up to 5 minutes to reduce rereading the file
@cache.memoize(engine="memcache", key="code.get_team_members", expires=5 * 60)
def get_team_members() -> list[dict[str, str]]:
    pass  # <- put file read, decode, and sort logic here.


def GET(self):
        return render_template("about/index.html", team_members=get_team_members())
```

Reply:
I would be OK if we land this PR without this caching and then do a separate follow-up PR to add memcache.

I am never too clear on `plugin` vs. `core` separation but perhaps it would be best practice to create a `team.py` file in https://github.com/internetarchive/openlibrary/tree/master/openlibrary/core to contain the `get_team_members()` function.  Others can verify.